config/systemSmartContractsConfig.go

```diff
@@ -36,11 +36,18 @@ type ESDTSystemSCConfig struct {
 
 // GovernanceSystemSCConfig defines the set of constants to initialize the governance system smart contract
 type GovernanceSystemSCConfig struct {
-	ProposalCost     string
-	MinQuorum        string
-	MinPassThreshold string
-	MinVetoThreshold string
-	EnabledEpoch     uint32
+	ProposalCost       string
+	ProposalCostV2     string
```

Comment:
Is this scalable? What happens when we will have V3, V4....?

Reply:
^ same as above, I went the same way as for other configs. I could change it, and inject a new config everywhere (or try a trick with struct composition so it would be easier for future versions), but we need to do it the same way everywhere then

Reply:
done, kept V1 and Active as namings